redshift/redshift.go

```diff
@@ -62,6 +62,20 @@ type ColInfo struct {
 	SortOrdinal int    `yaml:"sortord"`
 }
 
+type rangeQuery int
+
+const (
+	rangeNone rangeQuery = iota
```

Comment:
`rangeNone` means that we scan the entire table, right? I would maybe call that `rangeAll`.

Reply:
I like that. Updated.